app/main.py

```diff
@@ -1 +1,33 @@
-# write your code here
+from typing import List
+
+
+class Animal:
+    alive: List["Animal"] = []
+
+    def __init__(self, name: str, health: int = 100,
+                 hidden: bool = False) -> None:
+        self.name: str = name
+        self.health: int = health
+        self.hidden: bool = hidden
+        self.__class__.alive.append(self)
+
+    def __str__(self) -> str:
+        return f"{{Name: {self.name}, " \
+            f"Health: {self.health}, Hidden: {self.hidden}}}"
```

Comment:
better to use `()` then `\`

Reply:
Please provide an example of the linebreak with (). Thank you!